assets/prefabs/Igloo.prefab

```diff
@@ -0,0 +1,225 @@
+            { "blockType": "engine:air", "region": { "min": [5, 4, 8], "size": [2, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [8, 4, 8], "size": [6, 2, 1]}},
+            { "blockType": "engine:air", "region": { "min": [10, 3, 8], "size": [4, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [11, 1, 8], "size": [3, 2, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 1, 9], "size": [2, 2, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 3, 9], "size": [4, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 4, 9], "size": [5, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 5, 9], "size": [14, 2, 1]}},
+            { "blockType": "engine:air", "region": { "min": [4, 2, 9], "size": [5, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [5, 1, 9], "size": [4, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [5, 3, 9], "size": [2, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [8, 4, 9], "size": [6, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [9, 3, 9], "size": [5, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [10, 1, 9], "size": [4, 2, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 1, 10], "size": [2, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 2, 10], "size": [3, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 3, 10], "size": [5, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 4, 10], "size": [14, 3, 2]}},
+            { "blockType": "engine:air", "region": { "min": [7, 3, 10], "size": [7, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [9, 1, 10], "size": [5, 2, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 1, 11], "size": [3, 2, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 3, 11], "size": [4, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [5, 3, 11], "size": [9, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [6, 2, 11], "size": [8, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [9, 1, 11], "size": [5, 1, 1]}},
+            { "blockType": "engine:air", "region": { "min": [0, 1, 12], "size": [14, 6, 1]}}
+        ]
+    },
+    "AddItemsToChest": {
+        "chestsToFill": [
+            {
+                "position": [4, 1, 9],
+                "items": [
+                        {"slot": 0, "blockFamiliy": "CoreAssets:IronBar", "amount": 49},
+                        {"slot": 1, "blockFamiliy": "CoreAssets:GoldBar", "amount": 17},
+                        {"slot": 2, "blockFamiliy": "CoreAssets:DiamondBar", "amount": 3}
+                ]
+            }
+        ]
+    }
```

Comment:
No..... that's not a bug.. I think there must be some extra piece of logic in ST for placing these items inside there, and since I only extracted the structure building part from there, it doesn't work here,  because I'm pretty sure they were there when I placed it using the spawner from toolbox.. I'll try to find it....